---
title: 'Modeling single cell RNAseq data with multinomial distribution '
date: '2019-11-26'
linkTitle: https://divingintogeneticsandgenomics.rbind.io/post/modeling-single-cell-rnaseq-data-with-multinomial-distribution/
source: DNA confesses Data speak on DNA confesses Data speak
description: |-
  <p>I was reading <a href="https://www.biorxiv.org/content/10.1101/574574v1">Feature Selection and Dimension Reduction for Single Cell RNA-Seq based on a Multinomial Model</a>. In the paper, the authors model the scRNAseq counts using a multinomial distribution.</p>
  <p><img src="https://divingintogeneticsandgenomics.rbind.io/img/multinomial.png" /></p>
  <p>I was using negative binomial distribution for modeling in my last <a href="https://divingintogeneticsandgenomics.rbind.io/post/negative-bionomial-distribution-in-single-cell-rnaseq/">post</a>, so I asked the question on twitter:</p>
  <blockquo ...
disable_comments: true
---
<p>I was reading <a href="https://www.biorxiv.org/content/10.1101/574574v1">Feature Selection and Dimension Reduction for Single Cell RNA-Seq based on a Multinomial Model</a>. In the paper, the authors model the scRNAseq counts using a multinomial distribution.</p>
<p><img src="https://divingintogeneticsandgenomics.rbind.io/img/multinomial.png" /></p>
<p>I was using negative binomial distribution for modeling in my last <a href="https://divingintogeneticsandgenomics.rbind.io/post/negative-bionomial-distribution-in-single-cell-rnaseq/">post</a>, so I asked the question on twitter:</p>
<blockquo ...